.github/CODEOWNERS

```diff
@@ -1 +1 @@
-/terraform/     @tomtucka @sixdaysandy @xaithe
+/terraform/     @ministryofjustice/opg-webops
```

Comment:
I think we should include the UAL team in all too

Reply:
sure - happy to :)